src/main/java/cubicchunks/CommonEventHandler.java

```diff
@@ -45,7 +45,7 @@ public void onWorldLoad(WorldEvent.Load evt) {
 		ICubicWorld world = (ICubicWorld) evt.getWorld();
 
 		WorldType type = evt.getWorld().getWorldType();
-		if(type instanceof ICubicWorldType){
+		if(type instanceof ICubicWorldType) {
```

Comment:
Missing a space: `if(...)`. I suggest you configure your IDE's autoformatter and just let it format each file once.

[Here](https://stackoverflow.com/questions/36432438/importing-code-style-formatting-settings-into-eclipse-from-intellij-idea) seems to be some information on getting intellij settings into eclipse?